README.md

```diff
@@ -202,75 +224,16 @@ Models from https://github.com/onnx/models that have been tested to work with th
 
 
 ### Operators
-List of ONNX operators that can be converted into their CoreML equivalent:
-
-- Abs
-- Add
-- ArgMax
-- ArgMin
-- AveragePool
-- BatchNormalization
-- Clip
-- Concat
-- Conv
-- ConvTranspose 
-- DepthToSpace
-- Div
-- Elu
-- Exp
-- Flatten
-- Gemm
-- GlobalAveragePool 
-- GlobalMaxPool 
-- HardSigmoid
-- InstanceNormalization
-- LeakyRelu
-- Log
-- LogSoftmax
-- LRN
-- MatMul
-- Max
-- MaxPool 
-- Mean
-- MeanVarianceNormalization
-- Min
-- Mul
-- Neg
-- Pad
-- PRelu
-- Reciprocal
-- ReduceL1
-- ReduceL2
-- ReduceLogSum
-- ReduceMax
-- ReduceMean
-- ReduceMin
-- ReduceProd
-- ReduceSum
-- ReduceSumSquare
-- Relu
-- Reshape
-- Selu
-- Sigmoid
-- Slice
-- Softmax
-- Softplus
-- Softsign
-- SpaceToDepth
-- Split
-- Sqrt
-- Sub
-- Sum
-- Tanh
-- ThresholdedRelu
-- Transpose
-- Upsample
-
-Some of the operators are partially compatible because CoreML does not support gemm for arbitrary tensors, has limited support for non 4-rank tensors etc.   
+List of [ONNX operators supported in CoreML 2.0](https://github.com/onnx/onnx-coreml/blob/4d8b1cc348e2d6a983a6d38bb6921b6b77b47e76/onnx_coreml/_operators.py#L1893)
+
+List of [ONNX operators supported in CoreML 3.0](https://github.com/onnx/onnx-coreml/blob/4d8b1cc348e2d6a983a6d38bb6921b6b77b47e76/onnx_coreml/_operators_nd.py#L1821)
+
+
+Some of the operators are partially compatible because CoreML does not support gemm for arbitrary tensors.
```

Comment:
Some of the operators are partially compatible with Core ML, for example gemm with more than 1 non constant input is not supported in Core ML 2, or scale as an input for upsample layer is not supported in Core ML 3.